### PR TITLE
Add StartNewTopUpWithLimitsAndCustomToken

### PR DESCRIPTION
**Pull request by katamarinaki**
### Description

This PR adds a variation of `StartNewTopUpWithLimits` form called `StartNewTopUpWithLimitsAndCustomToken`. This form will allow operators to create `StartNewTopUpWithLimits` motions with the custom token select from tokens list approved by new `allowedTokensRegistry`.

[Link to Linear](https://linear.app/lidofi/issue/DAO-152/add-token-field-to-startnewtopupwithlimits-form)

### Checklist:

- [x] Checked the changes locally.
- [ ] Affects / requires changes in other services (Matomo / Sentry / CloudFlare / etc.).

### CODE REVIEW

File: modules/motions/ui/MotionFormStartNew/Parts/StartNewTopUpWithLimitsAndCustomToken.tsx [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+
+                <Fieldset>
+                  <InputNumberControl
+                    label={`${selectedTokenLabel} Amount`}
+                    name={`${fieldNames.programs}.${fieldIdx}.amount`}
+                    disabled={!selectedTokenAddress}
+                    rules={{
+                      required: 'Field is required',
+                      validate: value => {
+                        const check1 = validateToken(value)
```
Comment (BATMAH69) [inline modules/motions/ui/MotionFormStartNew/Parts/StartNewTopUpWithLimitsAndCustomToken.tsx:273]: could you give verbose name for `check1`

**Review by BATMAH69 — CHANGES_REQUESTED** [review]

File: modules/motions/types.ts [hunk 2]
```diff
@@ -10,9 +10,9 @@ export const MotionTypeForms = {
   AllowedRecipientTopUpTrpLdo: 'AllowedRecipientTopUpTrpLdo',
   LegoLDOTopUp: 'LegoLDOTopUp',
   LegoDAITopUp: 'LegoDAITopUp',
-  RccDAITopUp: 'RccDAITopUp',
-  PmlDAITopUp: 'PmlDAITopUp',
-  AtcDAITopUp: 'AtcDAITopUp',
+  RccStablesTopUp: 'RccStablesTopUp',
```
[the remark this answers is not in the document]
Comment (katamarinaki) [reply modules/motions/types.ts:13]: Yes, thank you for noticing!

File: modules/motions/ui/MotionFormStartNew/Parts/StartNewTopUpWithLimitsAndCustomToken.tsx (hunk 1 above)
Comment (katamarinaki) [reply modules/motions/ui/MotionFormStartNew/Parts/StartNewTopUpWithLimitsAndCustomToken.tsx]: Done!

File: modules/motions/hooks/useRegistryWithLimits.ts [hunk 3]
```diff
@@ -28,9 +28,9 @@ type AllowedRecipient = {
 export const REGISTRY_WITH_LIMITS_BY_MOTION_TYPE = {
   [MotionType.LegoLDOTopUp]: ContractLegoLDORegistry,
   [MotionType.LegoDAITopUp]: ContractLegoDAIRegistry,
-  [MotionType.RccDAITopUp]: ContractRccDAIRegistry,
-  [MotionType.PmlDAITopUp]: ContractPmlDAIRegistry,
-  [MotionType.AtcDAITopUp]: ContractAtcDAIRegistry,
+  [MotionType.RccDAITopUp]: ContractRccStablesRegistry,
```
Comment (BATMAH69) [inline modules/motions/hooks/useRegistryWithLimits.ts:31]: probaly should be old regestry 
```
  [MotionType.RccDAITopUp]: ContractRccDAIRegistry,
  [MotionType.PmlDAITopUp]: ContractPmlDAIRegistry,
  [MotionType.AtcDAITopUp]: ContractAtcDAIRegistry, 
```

Comment (katamarinaki) [reply modules/motions/hooks/useRegistryWithLimits.ts:31]: No, shouldn't be. New factories are working with old registries, so I think the name should be more generic.

**Review by BATMAH69 — APPROVED** [review]

**Pull request merged**